src/usr/bin/google_set_multiqueue

```diff
@@ -124,6 +124,24 @@ function is_gvnic() {
   return 0
 }
 
+# Returns the vCPU ranges on each of the numa nodes based on the platform.
```

Comment:
Why hard-code these ranges? Can you use the ranges as reported by `/sys/devices/system/node/node*/cpu*` or other sources of NUMA information instead?

Reply:
this path doesn't seem to list the vCPU range, it contains cpu<0...55>, I'm not quite sure how I can get the vCPU range based on this.
![image](https://github.com/user-attachments/assets/03e978ae-a872-4180-9adb-09b778445772)

Reply:
Looks like you might want `/sys/devices/system/node/node*/cpulist` if you want them as ranges.

Reply:
this only gives the first range, each numa should have 2 CPU ranges:
![image](https://github.com/user-attachments/assets/b3ecc2f9-2eeb-441c-bd20-e769bba0c3f3)

Reply:
Thanks for helping us spot this bug, seems it was already addressed in the last release and I was just running on a stale VM. Updated `get_vcpu_ranges` to read from sys file.